pkg/lvmd/cmd/root.go

```diff
@@ -79,6 +79,7 @@ func subMain() error {
 	for _, dc := range config.DeviceClasses {
 		_, err := command.FindVolumeGroup(dc.VolumeGroup)
 		if err != nil {
+			log.Error("Volume group not found:", dc.VolumeGroup);
```

Comment:
The last semicolon causes lint error, would you remove this?

I found another issue that the build is failed.
```
pkg/lvmd/cmd/root.go:82:43: cannot use dc.VolumeGroup (type string) as type map[string]interface {} in argument to "github.com/cybozu-go/log".
```

`log.Error` requires map but you specified string.
Could you revise this and test locally?